pkg/controller/statusmanager/status_manager_test.go

```diff
@@ -1188,3 +1208,171 @@ func setLastPodState(t *testing.T, client client.Client, name string, ps podStat
 		t.Fatal(err)
 	}
 }
+
+func TestStatusManagerCheckCrashLoopBackOffPods(t *testing.T) {
```

Comment:
Not really critical but having a test with the NonCriticalAnnotation would be a nice to have.

Reply:
yea, I'll add one with that, more coverage is always good.